src/lib/components/primitives/alert/alert.svelte

```diff
@@ -41,6 +41,7 @@
   )} tw_alert dark:tw_alert_dark tw_input-spacing tw_outline-none`}
   {id}
   tabindex="-1"
+  role={type === 'error' ? 'alert' : ''}
```

Comment:
This improves the accessibility of the alert message. If it's an error, the `role="alert"` more assertively notifies the user of the issue when using a screen reader.